src/test/java/core/basesyntax/strategy/AddOperationHandlerTest.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax.strategy;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.dto.Transaction;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.Operation;
+import org.junit.Test;
+
+public class AddOperationHandlerTest {
+    private final OperationHandler addHandler = new AddOperationHandler();
+    private final Fruit apple = new Fruit("apple");
+    private final Fruit orange = new Fruit("orange");
+
+    private static void makeStorageEmpty() {
+        Storage.storage.clear();
+    }
```

Comment:
```suggestion
    @Before
    public void tearDown {
        Storage.storage.clear();
    }
```
It will apply your logic before every test